util/new_project.sh

```diff
@@ -2,8 +2,10 @@
 # Script to make a new quantum project
 # Jack Humbert 2015
 
-if [ -z "$1" ]; then
-	echo "Usage: $0 <keyboard_name>"
+if [ -z "$1"  -o  -z "$2" ]; then
```

Comment:
This should default to AVR so we don't break current behavior. I think you could do that by moving the `KEYBOARD=$1` and `KEYBOARD_TYPE=$2` lines above this, then doing something like:

```
if [ -z "$KEYBOARD" -o "$KEYBOARD_TYPE" ]; then
   # Usage
    exit 1
elif [ -z "$KEYBOARD_TYPE" ]; then
  KEYBOARD_TYPE=avr
fi
```